services/ffc-demo/base/kustomize.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: ffc-demo-base
+  namespace: config-cluster-flux
+spec:
+  sourceRef:
+    kind: GitRepository
+    name: services-repository
+    namespace: config-cluster-flux
+  interval: 5m
+  timeout: 3m
+  prune: true
+  path: ./services/ffc-demo/${ENVIRONMENT}
+  postBuild:
+    substitute:
+      NAMESPACE: "ffc-demo"
+      WORKLOAD_IDENTITY_ID: "3acd7d69-86fb-40ce-a389-4632e864e425"
```

Comment:
where does this come from?

Reply:
@DaRoszko : This is a User MI Id which we setup manually and its hardcoded here for now to just test Service account changes. We will refine it when we will introduce service operators.